### DISCUSSION TITLE
Unflagged my account

### DISCUSSION THREAD

**Discussion by DarcShad** [community/community#50010] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello 
Please what should I do to unflagged my GitHub account?

**Comment by mrepol742** [comment]
Duplicate of
https://github.com/orgs/community/discussions/27294
the answer is
Contact the github support through this link:
https://support.github.com/contact/account

**Comment by tuves** [comment]
Hello, thank you for checking in. This account and topic has been flagged as: >One person or legal entity may maintain no more than one free Account.

Our private support team is the only one able to help with support requests on account flags. You will need to follow up with them directly through your open ticket.
